### PR TITLE
Chapter 6: MicroProfile Health

### PR DESCRIPTION
**Pull request by ttelang**
Adding Chapter 6 about MicroProfile Health

### CODE REVIEW

**Comment by Emily-Jiang** [comment]
@xstefank please review

**Comment by ttelang** [comment]
Awaiting review feedback and approval.

**Comment by ttelang** [comment]
Awaiting review feedback and approval.

File: chapter06/chapter06.adoc [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+where automatic scaling, failover, and recovery are essential for maintaining service availability and reliability. In this 
+chapter, we will learn about different types of health checks and standard health indicators provided by MicroProfile.
+
+=== Topics to be covered:
+
+- Overview of MicroProfile Health
+- Types of Health Checks
+- Standard Health Checks
+- Implementing and Exposing Health Checks
+- Logging and Reporting Health Checks
```
Comment (xstefank) [inline chapter06/chapter06.adoc:16]: You removed this section.

**Comment by xstefank** [comment]
@ttelang otherwise it looks good, so once you fix those two minor comments feel free to proceed. Good job!

File: chapter06/chapter06.adoc [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+need to make decisions about whether to restart a failing service, reroute traffic away from an unhealthy service, or take 
+other actions to maintain overall system reliability.
+  
+Let’s delve into the essentials of MicroProfile Health, its importance, and how it works.
+
+=== Key Concepts
+
+At its core, the MicroProfile Health specification defines a mechanism for microservices to report their health status via HTTP. 
+These health checks can be used by external systems to verify the operational status of the services. This is crucial in modern cloud 
+environments where automated processes.
```
[the remark this answers is not in the document]
Comment (ttelang) [reply chapter06/chapter06.adoc]: Fixed

File: chapter06/chapter06.adoc (hunk 1 above)
Comment (ttelang) [reply chapter06/chapter06.adoc]: Updated the Toc as per sections of this chapter.

**Comment by ttelang** [comment]
@Emily-Jiang - Resolved all the comments, kindly merge this PR.

**Pull request merged**